src/main/steps/common/task-list/content.test.ts

```diff
@@ -82,29 +82,29 @@ describe('testcase for tasklist', () => {
       },
     ],
     partyName: 'undefined undefined',
-    progressBar: [
-      {
-        ariaLabel: 'Application submitted stage is not yet started',
-        label: 'Application<br/> submitted',
-        statusBarClassName: '',
-      },
-      {
-        ariaLabel: 'Cafcass child safety checks stage is not yet started',
-        label: 'Cafcass child<br/> safety checks',
-        statusBarClassName: '',
-      },
-      {
-        ariaLabel: 'Response submitted stage is not yet started',
-        label: 'Response<br/> submitted',
-        statusBarClassName: '',
-      },
-      {
-        ariaLabel: 'Hearings and court orders stage is not yet started',
-        label: 'Hearings and<br/> court orders',
-        statusBarClassName: '',
-      },
-      { ariaLabel: 'Case closed stage is not yet started', label: 'Case closed', statusBarClassName: '' },
-    ],
+    // progressBar: [
```

Comment:
please remove commented line of code.

Reply:
added LD flag, so we don't need to comment out portion of the code for future